main.py

```diff
@@ -41,11 +45,16 @@ def get_stats():
     for l in lang_data[:5]:
         ln = len(l['name'])
         ln_text = len(l['text'])
-        fmt_percent = format(l['percent'], '0.2f').zfill(5) # to provide a neat finish.
-        data_list.append(
-            f"{l['name']}{' '*(12-ln)}{l['text']}{' '*(20-ln_text)}{make_graph(l['percent'])}   {fmt_percent} %")
+        op = f"{l['name']}{' '*(12-ln)}{l['text']}{' '*(20-ln_text)}{make_graph(l['percent'])}   {l['percent']}%"
```

Comment:
Opps! I think you removed the pretty formatting. This is from the old file...

Reply:
Oh Damn🤦‍♂️